appserver/tests/appserv-tests/devtests/deployment/run_test.sh

```diff
@@ -169,5 +169,7 @@ case $OPT in
     list_test_ids;;
   run_test_id )
     run_test_id $TEST_ID ;;
+  deployment_all )
```

Comment:
I added for easy to run the tests locally. 
No, run_test_id is doing some intialization which I don't want locally so just added target, I set S1AS & APS, it works fine so added.